onyo/commands/cat.py

```diff
@@ -60,7 +60,8 @@ def cat(args: argparse.Namespace, opdir: str) -> None:
     """
     repo = None
     try:
-        repo = Repo(opdir)
+        repo_root = Repo.find_root(opdir)
```

Comment:
IMO, I envision `_find_root()` as a private method for `Repo`, and the user does not interact with it directly.

Instead the syntax would be like:
```
repo = Repo(opdir, search=True)
```
Then if they want to know what the repo root is, they do it they same was as always:
```
repo.root
```

The name `search` could be better. It's just what came to mind in this moment.